vividus-plugin-azure-cosmos-db/src/main/java/org/vividus/azure/cosmos/CosmosDbService.java

```diff
@@ -47,12 +48,21 @@ public CosmosContainer load(CosmosDbContainer cosmosDbContainer)
                         String accountKey = db.getAccountKey();
                         CosmosDbAccount account = accounts.get(accountKey,
                                 "Account configuration not found for the key: %s", accountKey);
-                        return new CosmosClientBuilder().endpoint(account.getEndpoint())
-                                                        .key(account.getKey())
-                                                        .connectionSharingAcrossClientsEnabled(true)
-                                                        .buildClient()
-                                                        .getDatabase(db.getId())
-                                                        .getContainer(cosmosDbContainer.getId());
+
+                        String key = account.getKey();
+                        CosmosClientBuilder builder = new CosmosClientBuilder().endpoint(account.getEndpoint())
+                                .connectionSharingAcrossClientsEnabled(true);
+                        if (key == null)
+                        {
```

Comment:
add logging: `info("Accessing Cosmos DB '{}' using default credentials, as no key has been configured for it", accountKey)